src/folsomite_server.erl

```diff
@@ -182,6 +188,10 @@ get_system_info() ->
     [{K, V} || {K, V} <- L,
                lists:member(K, [port_count, process_count])].
 
+truncate_metric_name(Metric, Length) ->
+    Format = lists:flatten(io_lib:format("~~-~Bs~n", [Length])),
```

Comment:
Better way to do the runtime sized string truncation with io_lib:format?

Reply:
Seems reasonable to me

Reply:
Should this be `Length - 1` because of the `\n`?

Reply:
Great point! Can change